resources/ext/build-scripts/install-dropsonde.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+FORGE_API="forgeapi-cdn.puppet.com"
+MODULE_SLUG="puppetlabs-dropsonde"
+MODULE_NAME="dropsonde"
+MODULE_VERSION="0.0.5"
+
+curl -O "https://${FORGE_API}/v3/files/${MODULE_SLUG}-${MODULE_VERSION}.tar.gz"
+mkdir "${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules"
+tar -xvf "${MODULE_SLUG}-${MODULE_VERSION}.tar.gz" -C "${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules"
+mv "${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules/${MODULE_SLUG}-${MODULE_VERSION}" "${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules/${MODULE_NAME}"
```

Comment:
What will happen here on upgrades? I think we might need to remove the unversioned `$MODULE_NAME` directory before we can move the newly-installed version over.

Reply:
that step is required when a module is installed.
This script reproduces `puppet module install puppetlabs-dropsonde --version 0.0.5`
How does the upgrade look like? It's a script?
If it is a script that runs over an installed puppetserver the command 
`/opt/puppetlabs/puppet/bin/puppet module install puppetlabs-dropsonde --modulepath /opt/puppetlabs/server/data/puppetserver/puppetserver_modules` should work.

Reply:
I think this comes down to how the package would handle differences in the module's files, right? Do you know how that works, Molly?

Reply:
@Magisus what do you mean by 'how the package would handle differences in the module's files'?

My concern was around the `mv` command and how it behaves when the destination exists already. I'm pretty sure this will put the versioned module _into_ `$DESTDIR/opt/puppetlabs/server/data/puppetserver/puppetserver_modules/$MODULE_NAME`, rather than renaming it, if that directory exists already and is non-empty.